x-pack/plugins/synthetics/kibana.jsonc

```diff
@@ -33,7 +33,16 @@
       "usageCollection",
       "bfetch"
     ],
-    "optionalPlugins": ["cloud", "data", "fleet", "home", "ml", "spaces", "telemetry"],
+    "optionalPlugins": [
+      "cloud",
+      "data",
+      "fleet",
+      "home",
+      "ml",
+      "serverless",
```

Comment:
Do you still need the `serverless` dependency here?

Reply:
Perhaps not for this change, as we reference the config env fields now, but we will consume the breadcrumbs API in a future PR, and I believe without it the start plugin will not get injected into the app.